auto-spacing.txt

```diff
@@ -592,17 +592,20 @@
 2FF0..2FFB     ; O  # W   IDEOGRAPHIC DESCRIPTION CHARACTER LEFT TO RIGHT..IDEOGRAPHIC DESCRIPTION CHARACTER OVERLAID
 3000           ; O  # F   IDEOGRAPHIC SPACE
 3001..3004     ; O  # W   IDEOGRAPHIC COMMA..JAPANESE INDUSTRIAL STANDARD SYMBOL
-3005..3007     ; W  # W   IDEOGRAPHIC ITERATION MARK..IDEOGRAPHIC NUMBER ZERO
-3008..3020     ; O  # W   LEFT ANGLE BRACKET..POSTAL MARK FACE
-3021..302D     ; W  # W   HANGZHOU NUMERAL ONE..IDEOGRAPHIC ENTERING TONE MARK
-302E..302F     ; N  # W   HANGUL SINGLE DOT TONE MARK..HANGUL DOUBLE DOT TONE MARK
+3005           ; N  # W   IDEOGRAPHIC ITERATION MARK
+3006           ; W  # W   IDEOGRAPHIC CLOSING MARK
+3007..3029     ; O  # W   IDEOGRAPHIC NUMBER ZERO..HANGZHOU NUMERAL NINE
```

Comment:
[U+3007](https://codepoints.net/U+3007): Shouldn't this stay `W`, no?

Reply:
Yes, we discussed this at JLReq TF. As U+3007 is used together with Han numerals, it should be W.
U+3005 should also be W.